examples/pytorch/ogb/ngnn_seal/main.py

```diff
@@ -48,17 +48,16 @@ def __init__(
         else:
             self.node_features = None
 
-        if not self.dynamic:
-            self.g_list, tensor_dict = self.load_cached()
-            self.labels = tensor_dict["y"]
-            return
-
         pos_edge, neg_edge = get_pos_neg_edges(
-            split, self.split_edge, self.graph, self.percent
+            self.split, self.split_edge, self.graph, self.percent
         )
         self.links = torch.cat([pos_edge, neg_edge], 0)  # [Np + Nn, 2]
```

Comment:
Semantically, the logic of the code has changed, I am wondering why it was OK to return at L54? self.links was not set in the previous code. Was it a bug?

Reply:
not a bug. In the previous code, self.links would be set in method self.load_cached() at L52, and I remove the relevant code in that method (the same as L51-L54 of the new version).